common/src/main/scala/com/gu/media/Settings.scala

```diff
@@ -6,9 +6,9 @@ import scala.collection.JavaConverters._
 trait Settings {
   val config: Config
 
-  def getString(name: String): Option[String] = if(config.hasPath(name)) { Some(config.getString(name)) } else { None }
-  def getStringList(name: String): List[String] = if(config.hasPath(name)) { config.getStringList(name).asScala.toList } else { List.empty}
-  def getBoolean(name: String): Option[Boolean] = if(config.hasPath(name)) { Some(config.getBoolean(name)) } else { None }
+  def getString(name: String): Option[String] = if(config != null && config.hasPath(name)) { Some(config.getString(name)) } else { None }
```

Comment:
The trait stacking and overriding meant that it was null in test. I'll take another look at how to structure it, it's not a great fix and I ran into the same problem writing tests for the uploader lambda